plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java

```diff
@@ -298,7 +297,7 @@ private static StandardSQLTypeName toStandardSqlTypeName(Type type)
         if (type == TimestampWithTimeZoneType.TIMESTAMP_TZ_MICROS) {
             return StandardSQLTypeName.TIMESTAMP;
         }
-        if (type instanceof CharType || type instanceof VarcharType) {
+        if (type instanceof VarcharType) {
```

Comment:
why this change?

Reply:
It was required to fix incorrect predicates between `char` vs `varchar` if I remember correctly.

Reply:
What is CharType mapped to now? I don't see explicit mapping so does it mean it's unsupported type now?

Reply:
It's mapped to BigQuery `STRING` type now.

Reply:
I'm probably being stupid but I can't find code which handles that (mapping Trino char to BigQuery STRING). Can you point me to it?

I'm probably confused because of removal of `char` from `createTableSupportedTypes` in `TestBigQueryConnectorTests`.

Reply:
Sorry, I misunderstood "now" as "before this change". I wanted to say:
* `CHAR` was mapped to `STRING` before this PR
* `CHAR` is unsupported after this PR

Reply:
Sorry for causing confusion. 😄 Do we want to mention this in release notes? The only possible impact I see is some CTAS statements from other catalogs might fail but easy to preserve old "incorrect" behaviour by casting CHAR columns to VARCHAR instead.